helm/examples/logging/README.md

```diff
@@ -43,7 +43,13 @@ The availble target audiences include operator(OPR), user(USR), programmer(PRO),
 The available target categories include disaster(DIS), error(ERR), warning(WRN),information(INF),progress(PRO). The category (or class) filter is controlled by the `<section>`.logging.classes value, comprised of 3 letter codes delimited by the aggregation operator (+) or the removal operator (-).
     
     For example, dali component log output to include all classes except for progress
-    helm install myhpcc ./hpcc --set global.logging.classes="ALL-PRO"
+    helm install myhpcc ./hpcc --set dali.logging.classes="ALL-PRO"
+
+### Log Detail Level Configuration
+Log output verbosity can be adjusted from "critical messages only" (1) up to "report all messages" (100). By default, the log level is set highest (100). 
+
+    For example, ESP component log verbosity should be medium
+    helm install myhpcc ./hpcc --set esp.logging.detail="50"
```

Comment:
think this and the dali --set won't work, because dali, esp are lists.
Apart from the global default logging level, you'll probably have to advise modifying the sections in a copy of values.yaml